src/package.json

```diff
@@ -20,6 +21,11 @@
   "jest": {
     "testEnvironment": "node"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": ""
```

Comment:
@patrickpingol for now we will not run the linter before committing since there are a lot of lint errors in our code right now. Once we clean up each of our code we can implement this.

Reply:
I understand. But it'd take a lot of time if we wait for each other to fix lint errors. I'll try to make a PR tomorrow to fix it all once this gets merged and add `npm run lint` here. 👍

Reply:
I'll fix the lint errors 👍

Reply:
Will you include it in this PR? It'll be a huge changeset, I suggest we merge this and create a new branch for the lint fix.

Reply:
should we merge this first before I fix the errors?

Reply:
yeah i think it's better if we merge it first